lambdas/README.md

```diff
@@ -1,12 +1,64 @@
+
+A single version (PROD) of this lambda exists in AWS, and will rebuild automatically when a PR branch merges into `main`.
+
+### MembershipSub-PromoCode-View-Dynamo-to-Salesforce
+A Lambda function to replicate the MembershipSub-PromoCode-View-DEV Dynamo table to Salesforce.
+
+This lambda sends data to salesforce on a daily schedule. We can just check the lambda's cloudwatch logs to confirm that it's happy - it logs all the requests it's making.
+
+Three versions of this lambda (PROD, UAT, DEV) exist in AWS, each of which maps to thge equivalent environment in the SalesForce cloud. We can use the DEV version for testing. 
+
+The lambda relies on SalesForce usernames/passwords which have been stored in AWS as encrypted environment variables.
+
+Only the PROD will rebuild automatically when a PR branch merges into `main`.
+
+### MembershipSub-Promotions-Scheduled-Dynamo-Backup
+A Lambda function to trigger the backup of the Campaigns and Promotions table and delete any old backups (retention days is defined in the code).
+
+For dev work, we can manually run it in AWS and check it worked from the Dynamo page in the AWS console.
+
+A single version (PROD) of this lambda exists in AWS, and will rebuild automatically when a PR branch merges into `main`.
+
+### MembershipSub-Promotions-to-PromoCode-View
+An Amazon DynamoDB trigger that creates a view of data associated with a particular promo code for the data pipeline to collect.
+
+The code writes updates from the Promotions table to the PromoCode-View table. 
+
+During development work, we can test by creating a test promotion in the Promo Tool (this repo's frontend) in its DEV page and check that it gets written to PromoCode-View-DEV table.
+
+Three versions of this lambda (PROD, UAT, DEV) exist in AWS. We can use the DEV version for testing. Only the PROD will rebuild automatically when a PR branch merges into `main`.
+
+### MembershipSub-Reconstruct-PromoCode-View
+This lambda appears to be for rebuilding the MembershipSub-PromoCode-View table. The intention is for it to be run manually if that is ever required - https://github.com/guardian/memsub-promotions/blob/7009715e905aa68de3b5c939ccc54f97b13e3c68/lambdas/src/MembershipSub-Reconstruct-PromoCode-View.js#L4
+
+
+## Development
+Because these are lambdas, we cannot develop and test on local devices. Instead code needs to be written locally and then all the lambda files in the lambdas/source folder (but not the folder itself, or any subfolders) need to be zipped and uploaded manually - using the AWS lambdas console - for testing.
+
 
 ## Deployment
 
 The deploy is called MemSub::Membership Admin::Promotions Tool Lambda
 
 teamcity will
-1. run lambdas/build.sh
-1. then upload the MembershipSub-Promotions-to-PromoCode-View-Lambda.zip to the Lambda function on S3 
-1. refresh the package
++ run lambdas/build.sh
```

Comment:
I think we can remove this teamcity list, it's not accurate (riffraff updates the lambdas) and it's all pretty standard

Reply:
Removed TeamCity reference/actions